embedly-server.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+
+const connect = require("connect");
+const http = require("http");
+const query = require("connect-query");
+const cors = require("cors");
+const request = require("request");
+const yaml = require("yamljs");
+
+const API_URL = "https://api.embedly.com/1/extract";
+
+let config = yaml.load("config.default.yml");
+try {
+  // Load user config if it exists
+  config = Object.assign({}, config, yaml.load("config.yml"));
+
+} catch (e) {
+  //
+}
+
+if (!config.API_KEY) {
+  throw new Error("You must create a config.yml and add API_KEY for this server to run.");
+}
+
+console.log(config);
+
+const app = connect();
+app.use(cors());
+app.use(query());
+
+app.use("/extract", function(req, res) {
+  let qs = `?key=${config.API_KEY}`;
+  if (req.query.urls) {
+    qs += `&urls=${req.query.urls.map(encodeURIComponent).join(",")}`;
+  } else if (req.query.url) {
+    qs += `req.query.url${encodeURIComponent(req.query.url)}`;
```

Comment:
What is this line doing? It looks like it might bust the query string if you have a `config.API_KEY` and `req.query.url` defined.

For example, if you have `?key=ABC` and then `req.query.url` is defined (ie: "FOOBAR"), we'd potentially see something like `?key=ABCreq.query.urlFOOBAR`.
Not sure if this should be
```js
qs += `&url=${encodeURIComponent(req.query.url)}`;
```